server/alerts_email_via_smtp.py

```diff
@@ -0,0 +1,270 @@
+<td style='padding:.75pt .75pt .75pt .75pt;color:white'><p class=MsoNormal>Powered by: 
+<a href="https://github.com/securestate/king-phisher">RSM US, LLP</a> 
+<o:p></o:p></p></td></tr></table></div></td></tr></table><div class="Footer">
+<img src="https://github.com/securestate/king-phisher/raw/master/data/king-phisher-logo.png">
+</div>
+<p class=MsoNormal><o:p>&nbsp;</o:p></p></div></body></html>
+"""
+
+class Plugin(plugins.ServerPlugin):
+	authors = ['Austin DeFrancesco', 'Spencer McIntyre', 'Mike Stringer', 'Erik Daguerre']
+	classifiers = ['Plugin :: Server :: Notifications :: Alerts']
+	title = 'Campaign Alerts: via Python 3 SMTPLib'
+	description = """
+	Send campaign alerts via the SMTP Python 3 lib. This requires that users specify
+	their email through the King Phisher client to subscribe to notifications.
+	"""
+	homepage = 'https://github.com/securestate/king-phisher-plugins'
+
+	# Email accounts with 2FA, such as Gmail, will not work unless "less secure apps" are allowed
+	# Reference: https://support.google.com/accounts/answer/60610255
+	# Gmail and other providers require SSL on port 465, TLS will start with the activation of SSL
+	options = [
+		plugin_opts.OptionString(
+			name='smtp_host',
+			description='Location of SMTP server',
+			default='localhost'
+		),
+		plugin_opts.OptionInteger(
+			name='smtp_port',
+			description='Port used for SMTP server',
+			default=25
+		),
+		plugin_opts.OptionString(
+			name='smtp_email',
+			description='SMTP email address to send notifications from',
+			default=''
+		),
+		plugin_opts.OptionString(
+			name='smtp_password',
+			description='Password associated with smtp email',
+			default=''
+		),
+		plugin_opts.OptionBoolean(
+			name='ssl',
+			description='Set connection to smtp server to use ssl connection',
+			default=True
+		),
+		plugin_opts.OptionString(
+			name='email_jinja_template',
+			description='Password associated with smtp email',
+			default=''
+		),
+		plugin_opts.OptionString(
+			name='mail_options',
+			description='Additional mail options',
+			default=''
+		),
+	]
+	req_min_version = '1.10'
+
+	def initialize(self):
+		signals.campaign_alert.connect(self.on_campaign_alert)
+		email_jinja_template = ''
+		if os.path.isfile(self.config['email_jinja_template']):
+			with open(self.config['email_jinja_template'], 'r') as file_:
+				email_jinja_template = file_.read()
+		self.email_template = email_jinja_template if email_jinja_template else HTML_EMAIL_TEMPLATE
+		self.render_template = king_phisher.templates.TemplateEnvironmentBase().from_string(self.email_template)
+		return True
+
+	def on_campaign_alert(self, table, alert_subscription, count):
+		user = alert_subscription.user
+		if not self.config['smtp_email']:
+			self.logger.debug("user {0} has no email address specified, skipping SMTP alert".format(user.id))
+			return False
+		if self.send_message(table, alert_subscription, count, self.config['smtp_email']):
```

Comment:
Just return the result from `self.send_message`, the conditional block is useless here.